docs/designers-developers/developers/block-api/block-patterns.md

```diff
@@ -0,0 +1,35 @@
+# Patterns
+
```

Comment:
Let's add a disclaimer that this are is still under active development.

Reply:
Thinking of adding `__experimental` prefix as well.